t/062-count.t

```diff
@@ -172,7 +172,7 @@ n = 24
 --- request
 GET /test
 --- response_body
-n = 24
+n = 25
```

Comment:
We definitely need more test cases to actually cover the new code paths you added in this patch.